exercises/book-store/canonical-data.json

```diff
@@ -1,13 +1,15 @@
 {
   "exercise": "book-store",
-  "version": "1.2.0",
+  "version": "1.3.0",
   "cases": [
     {
       "description": "Return the total basket price after applying the best discount.",
       "comments": [
         "Calculate lowest price for a shopping basket containing books only from ",
         "a single series.  There is no discount advantage for having more than ",
-        "one copy of any single book in a grouping."
+        "one copy of any single book in a grouping.",
+        "implementors should use proper fixed-point or currency data types of the ",
```

Comment:
maybe it would be helpful to be explicit in saying that the values in the JSON file are numbers of cents?